### PR TITLE
Add a line on contribute doc to ask author publishing Microsoft authorized template to azure samples browser

### PR DESCRIPTION
**Pull request by hemarina**
The manual test team are constantly tracking and creating PRs to publish msft authorized templates to azure samples browser. We should consider letting the author decides if they want to add their templates on azure samples browser.

### CODE REVIEW

File: website/docs/contribute.md [hunk 1]
```diff
@@ -22,8 +22,9 @@ To submit a template:
     - **Author's Name** - Name to credit on the gallery card
     - **Link to template source** - Link to the template GitHub repo
     - **Tags** - One or more [tags](https://github.com/Azure/awesome-azd/blob/main/website/src/data/tags.tsx) representing the template. Provide at least 1 tag for programming language used and at least 1 tag for Azure services integrated. Also tag the IaC provider (Bicep or Terraform). If you don't see a relevant tag for your template? Feel free to add one!
-2. Open a PR!
-3. If possible, add a link to the PR in your repo where you made your app `azd` compatible to the PR description. This will help us provide feedback on your template and speed up the review process. 
+2. If the template is microsoft authorized, we encourage to [publish the template to azure samples browsers](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main).
```
Comment (kristenwomack) [inline website/docs/contribute.md]:
```suggestion
2. If the template is Microsoft authorized, we encourage to [publish the template to Azure samples browsers](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main). 
```

Comment (kristenwomack) [reply website/docs/contribute.md]: What does Microsoft authorized mean? Is this one that is authored by a Microsoft employee? Or part of a Microsoft curated collection of templates?

File: website/docs/contribute.md [hunk 2]
```diff
@@ -22,8 +22,9 @@ To submit a template:
     - **Author's Name** - Name to credit on the gallery card
     - **Link to template source** - Link to the template GitHub repo
     - **Tags** - One or more [tags](https://github.com/Azure/awesome-azd/blob/main/website/src/data/tags.tsx) representing the template. Provide at least 1 tag for programming language used and at least 1 tag for Azure services integrated. Also tag the IaC provider (Bicep or Terraform). If you don't see a relevant tag for your template? Feel free to add one!
-2. Open a PR!
-3. If possible, add a link to the PR in your repo where you made your app `azd` compatible to the PR description. This will help us provide feedback on your template and speed up the review process. 
+2. If the template is microsoft authorized, we encourage to [publish the template to azure samples browsers](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main). 
+3. Open a PR!
+4. If possible, add a link to the PR in your repo where you made your app `azd` compatible to the PR description. This will help us provide feedback on your template and speed up the review process.
```
Comment (kristenwomack) [inline website/docs/contribute.md:27]: I like this.

**Review by kristenwomack — CHANGES_REQUESTED** [review]
I suggested a change, and I have an open question on a point I don't understand. I'm supportive of the spirit of this change. We'll want to also look at this in the lens of the azd artifacts in the template standardization effort.

File: website/docs/contribute.md (hunk 1 above)
Comment (gkulin) [inline website/docs/contribute.md]:
```suggestion
2. If the template is Microsoft-authored, we encourage you to also [publish it to learn.microsoft.com/samples](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main). 
```

Comment (hemarina) [reply website/docs/contribute.md]: The template is hosted under Microsoft org like Azure-Samples, Azure, dotnet, etc. and maintained by Microsoft teams or employees.

**Review by kristenwomack — APPROVED** [review]
LGTM

**Review by gkulin — APPROVED** [review]

**Pull request merged**